hs/src/Reach/EPP.hs

```diff
@@ -428,6 +450,13 @@ be_m = \case
   where
     nop at = retb0 $ const $ return $ DL_Nop at
 
+multipleApiCalls :: SLPart -> BApp Bool
+multipleApiCalls who = do
+  apis <- asks be_api_calls
+  case M.lookup who apis of
+    Just n  -> return $ n > 1
+    Nothing -> return $ False
```

Comment:
This function is copied in all of these fields but using different environment accessors and the previous comment would make it irrelevant